src/AppBundle/Entity/SocialShareCategory.php

```diff
@@ -45,28 +44,28 @@ class SocialShareCategory
 
     public function __construct(string $name = '', int $position = 1)
     {
-        $this->name = EmojisRemover::remove($name);
+        $this->name = $name;
         $this->position = $position;
     }
 
-    public function __toString()
+    public function __toString(): string
     {
-        return $this->name ?: '';
+        return $this->name;
     }
 
     public function getId(): int
     {
         return $this->id;
     }
 
-    public function getName(): ?string
+    public function getName(): string
```

Comment:
I think this won't always work (when the form will put null data in the name property).